Content.Server/Actions/GhostBoo.cs

```diff
@@ -0,0 +1,44 @@
+#nullable enable
+using System.Linq;
+using Content.Server.GameObjects.Components.Observer;
+using Content.Shared.Actions;
+using Content.Shared.GameObjects.Components.Mobs;
+using Content.Shared.Utility;
+using JetBrains.Annotations;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.Actions
+{
+    /// <summary>
+    ///     Blink lights and scare livings
+    /// </summary>
+    [UsedImplicitly]
+    public class GhostBoo : IInstantAction
+    {
+        private float _radius;
+        private float _cooldown;
+
+        void IExposeData.ExposeData(ObjectSerializer serializer)
+        {
+            serializer.DataField(ref _radius, "radius", 10);
+            serializer.DataField(ref _cooldown, "cooldown", 10);
```

Comment:
i'd just bump this up to two minutes. logic being: i'd consider anything more than (inclusively) 1 spook / 10 seconds annoying, so setting it to two minutes would:
- defeat any real way to metagame as latency would be painfully long between spooks
- would require 12 players coordinating to spook at 1 spook / 10 secs
- also what sloth said this is somewhat expensive

that being sad this is SOUL and require to be in the game
thank you for your time